scripts/nightly-cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
Can we add `set -eou pipefail` here? (But _not_ `x`, as that would bleed secrets into the job logs!)

After that, this PR LGTM.

Reply:
Good call, added it